src/gamecontroller.cs

```diff
@@ -2170,5 +2202,485 @@ interface GCProductCategory
+
+		[Export ("snapshot")]
+		bool Snapshot { [Bind ("isSnapshot")] get; }
+
+		[Export ("capture")]
+		GCRacingWheel Capture { get; }
+
+		[Notification, Field ("GCRacingWheelDidConnectNotification")]
+		NSString GCRacingWheelDidConnectNotification { get; }
+
+		[Notification, Field ("GCRacingWheelDidDisconnectNotification")]
+		NSString GCRacingWheelDidDisconnectNotification { get; }
+	}
+
+	// TODO after I figure out xcode14 chat question
+	// [iOS (16,0), Mac (13,0), NoWatch, TV (16,0), MacCatalyst (16,0)]
+	// [BaseType (typeof (GCRacingWheelInputState))]
+	// interface GCRacingWheelInput : GCDevicePhysicalInput {
+	// 	[Export ("capture")]
+	// 	GCRacingWheelInputState Capture { get; }
+
+	// 	[NullAllowed, Export ("nextInputState")]
+	// 	IGCDevicePhysicalInputStateDiff NextInputState { get; }
+	// }
+
+	[iOS (16,0), Mac (13,0), NoWatch, TV (16,0), MacCatalyst (16,0)]
+	[BaseType (typeof (NSObject))]
+	interface GCRacingWheelInputState : GCDevicePhysicalInputState {
+		[Export ("wheel")]
+		GCSteeringWheelElement Wheel { get; }
+
+		[NullAllowed, Export ("acceleratorPedal")]
+		IGCButtonElement AcceleratorPedal { get; }
+
+		[NullAllowed, Export ("brakePedal")]
+		IGCButtonElement BrakePedal { get; }
+
+		[NullAllowed, Export ("clutchPedal")]
+		IGCButtonElement ClutchPedal { get; }
+
+		[NullAllowed, Export ("shifter")]
+		GCGearShifterElement Shifter { get; }
+	}
+
+	[iOS (16,0), Mac (13,0), NoWatch, TV (16,0), MacCatalyst (16,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface GCSteeringWheelElement : GCAxisElement {
+		[Export ("maximumDegreesOfRotation")]
+		float MaximumDegreesOfRotation { get; }
+	}
+
+	interface IGCAxisElementName {}
+
+	[iOS (16,0), Mac (13,0), NoWatch, TV (16,0), MacCatalyst (16,0)]
+	[Protocol]
+	interface GCAxisElementName : GCPhysicalInputElementName {
+	}
+
+	interface IGCButtonElementName {}
+
+	[iOS (16,0), Mac (13,0), NoWatch, TV (16,0), MacCatalyst (16,0)]
+	[Protocol]
+	interface GCButtonElementName : GCPhysicalInputElementName {
+	}
+
+	interface IGCPhysicalInputElementName {}
+
+	[iOS (16,0), Mac (13,0), NoWatch, TV (16,0), MacCatalyst (16,0)]
+	[Protocol]
+	interface GCPhysicalInputElementName {
+	}
+
+	interface IGCSwitchElementName {}
+
+	[iOS (16,0), Mac (13,0), NoWatch, TV (16,0), MacCatalyst (16,0)]
+	[Protocol]
+	interface GCSwitchElementName : GCPhysicalInputElementName {
+	}
+
+	[iOS (16,0), Mac (13,0), NoWatch, TV (16,0), MacCatalyst (16,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface GCPhysicalInputElementCollection<KeyIdentifierType, ElementIdentifierType> // : INSFastEnumeration // # no generator support for FastEnumeration - https://bugzilla.xamarin.com/show_bug.cgi?id=4391
+		where KeyIdentifierType : IGCPhysicalInputElementName /* NSString */ // there's currently not an conversion from GCPhysicalInputElementName, GCButtonElementName, and GCDirectionPadElementName to NSString
+		where ElementIdentifierType : IGCPhysicalInputElement
```

Comment:
I'm having trouble using this interface with all the different protocols that try to use different protocols for the 'KeyIdentifierType: GCPhysicalInputElementName, GCButtonElementName, and GCDirectionPadElementName